app/bundles/CampaignBundle/Assets/js/campaign.js

```diff
@@ -2330,3 +2333,45 @@ Mautic.previewCampaignLabels = function() {
         });
     });
 }
+
+Mautic.campaignAuditlogOnLoad = function (container, response) {
+    mQuery("#campaign-auditlog a[data-activate-details='all']").on('click', function() {
+        var $icon = mQuery(this).find('span').first();
```

Comment:
Let's use `const` and `let` (if necessary to reassign the variable) instead of `var` in the new JS code.